src/components/inputs/rich-text-input/icons/index.js

```diff
@@ -0,0 +1,45 @@
+import createStyledIcon from '../../../icons/create-styled-icon';
+import OrigBoldIcon from './svg/bold.react.svg';
+import OrigItalicIcon from './svg/italic.react.svg';
+import OrigOrderedListIcon from './svg/ordered-list.react.svg';
+import OrigMoreStylesIcon from './svg/more-styles.react.svg';
+import OrigUnderlineIcon from './svg/underline.react.svg';
+import OrigUnorderedListIcon from './svg/unordered-list.react.svg';
+import OrigStrikethroughIcon from './svg/strikethrough.react.svg';
+import OrigSuperscriptIcon from './svg/superscript.react.svg';
+import OrigSubscriptIcon from './svg/subscript.react.svg';
+import OrigRedoIcon from './svg/redo.react.svg';
+
```

Comment:
Nitpick: we should probably have this file generated, like we do with the other icons

Reply:
I'd rather wait until we get the rest of the required icons in though